### ISSUE TITLE
Redo 314.rb

### ISSUE THREAD

**Issue by prework-instructor-kevin** [TehyaB/actualize-exercises#6]
https://github.com/TehyaB/actualize-exercises/blob/ac05546c338b4606135513a40af8c5bbd881af7c/314.rb#L1-L38

Try limiting yourself to solving smaller problems that will get you pointed in the right direction. This week we want to focus on decomposing theses problems versus trying to solve them. 

EX: 
1 - Write a program that asks the user to enter a word, then print that word

2 - Write a program that asks the user to enter a word, then print that word with each character split into an array.

 In other words, show your work in steps.
